src/components/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {View} from 'react-native';
+import {Confirm} from '@atb/assets/svg/mono-icons/actions';
+import {StyleSheet} from '@atb/theme';
+
+type CheckedProps = {
+  checked: boolean;
+};
+
+export const SavedCheckbox: React.FC<CheckedProps> = ({checked}) => {
+  const styles = useStyles();
+  return (
+    <View
+      style={[
```

Comment:
Should add `accessibilityRole` and such on this component. The containers wrapping this component as of now have the accessibility setup, but I think now that this Checkbox is a resusable component under `src/components` it should be made good enough to be a "standalone" checkbox. Maybe add it to the DesignSystem screen as well?